src/lib/build/mod.rs

```diff
@@ -187,5 +189,14 @@ pub fn compile_bin(
     // The output exectable will always go in target/bin
     codegen_invoke.exec(bcx)?;
 
-    Ok(layout.bin.join(bin_target.name.clone()))
+    // Find the codegen executable since different backend
```

Comment:
From my testing with the javascript backend, this doesn't apply; codegen backends won't automatically add extensions to a given output name (this is why in all the examples for [codegen targets in the Idris doc](http://docs.idris-lang.org/en/latest/reference/codegen.html), the extension is added manually).

Reply:
But on Window the c backend add '.exe' extension to the target output. Maybe I can add a pattern that matches the output file with no addtional extension.